doozer/doozerlib/cli/images_health.py

```diff
@@ -66,7 +112,8 @@ def url_text(url, text):
     latest_success_bi_dt = ''
 
     for idx, record in enumerate(records):
-        # record=( jobid, state, timestamp, joburl)
+        # Generate the art-dash link for each record
+        art_dash_link = generate_art_dash_history_link(image, record, runtime)
```

Comment:
why we need to get all records link? I think this loop is used for calculate index